internal/client/deployments.go

```diff
@@ -60,29 +60,30 @@ func (c *DeploymentsClient) Create(ctx context.Context, data api.DeploymentCreat
 	return &deployment, nil
 }
 
-// List returns a list of Deployments based on the provided list of names.
-func (c *DeploymentsClient) List(ctx context.Context, _ []string) ([]*api.Deployment, error) {
+// Get returns details for a Deployment by ID.
+func (c *DeploymentsClient) Get(ctx context.Context, deploymentID uuid.UUID) (*api.Deployment, error) {
```

Comment:
This diff shows things in kind of a weird way. I technically replaced `List` with `GetByName`.

So to clarify:
- ` Get` is unchanged
- `GetByName` is added
- `List` is removed